lesson9.tex

```diff
@@ -0,0 +1,149 @@
+\documentclass{lessonplan}
+
+% Constants
+\newcommand{\lessonNumber}{9}
```

Comment:
This should be Cipher lesson, instead of 9. Maybe it's worth changing the name of this command from `lessonNumber` to `lessonShortName` or something. Also, note that would require changing the file directory with the images